### PR TITLE
Added Zendesk desktop layout json file. This will be used by our customers who uses the Zendesk crm connector application

### PR DESCRIPTION
**Pull request by gvallath**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Please check if the PR fulfills these requirements
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

#### (Bug fix, feature, docs update, ...)

`[Type here]`

#### (You can link to an open issue)

`[Type here]`

#### (if this is a feature change)?

`[Type here]`

#### (What changes might users need to make in their application due to this PR?)

`[Type here]`

## Other information:

`[Type here]`

### CODE REVIEW

**Comment by gvallath** [comment]
Added Zendesk desktop layout json file. This will be used by our customers who uses the Zendesk crm connector application

**Review by merajcal — APPROVED** [review]

**Review by babuthambidurai — APPROVED** [review]

**Comment by gvallath** [comment]
We have added crm connector desktop layout for JIRA ticket CX -6589 work around.

**Review by merajcal — APPROVED** [review]

File: Examples/Layouts/MSDynamics_CX-6589_workaround.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+    "version": "0.0.2",
+    "appTitle": "Contact Center Desktop",
```
Comment (merajcal) [inline Examples/Layouts/MSDynamics_CX-6589_workaround.json:3]: can you please change the title to "Webex Contact Center", this is now being the default title inside default layout

**Review by merajcal — CHANGES_REQUESTED** [review]

**Review by merajcal — APPROVED** [review]

File: Examples/Layouts/MSDynamics_CX-6589_workaround.json (hunk 1 above)
Comment (gvallath) [reply Examples/Layouts/MSDynamics_CX-6589_workaround.json]: Added the changes.

**Pull request merged**